src/test/jtx/impl/Env.cpp

```diff
@@ -537,6 +537,14 @@ Env::do_rpc(std::vector<std::string> const& args)
     return response;
 }
 
+void
+Env::enableFeature(uint256 const feature)
```

Comment:
I'm not a fan of adding this function here, particularly with the `close()` calls it makes. The ledger advances just by calling this method, which is not obvious. For the one place that you need this, I would just call the `features.insert(feature)` directly in the test. Also worth nothing that this test will need adjusting when we merge https://github.com/ripple/rippled/pull/2108 since that changes the default disposition of `Env` with respect to features.